src/ucs/config/global_opts.c

```diff
@@ -162,6 +163,9 @@ static ucs_config_field_t ucs_global_opts_table[] = {
    "Maximal size of profiling log. New records will replace old records.",
    ucs_offsetof(ucs_global_opts_t, profile_log_size), UCS_CONFIG_TYPE_MEMUNITS},
 #endif
+  {"PROXIMITY_PRIO", "y",
+   "Consider PCI proximity when looking for HCA.\n",
```

Comment:
this should be put in IB memory domain configuration, not if UCS (which is a generic infra library)